src/coreclr/tools/Common/JitInterface/CorInfoBase.cs

```diff
@@ -10,10 +10,10 @@
 
 namespace Internal.JitInterface
 {
-    unsafe partial class CorInfoImpl
+    internal unsafe partial class CorInfoImpl
```

Comment:
This is auto-generated file. Could you please fix the auto-generator at src/coreclr/tools/Common/JitInterface/ThunkGenerator as well ?